### PR TITLE
Sticky "paid for" band

### PR DESCRIPTION
**Pull request by regiskuckaertz**
After the top banner, now is the turn of the paid for band that sits along the top both on desktop and mobile.

paidfor-sticky

paidfor-sticky-mobile

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
["Paid for" band using CSS-based sticky](https://trello.com/c/SPbPS9Jz/83-paid-for-band-using-css-based-sticky)

File: static/src/javascripts-legacy/bootstraps/commercial.js [hunk 1]
```diff
@@ -72,15 +72,12 @@ define([
...
         ['cm-fill-advert-slots', fillAdvertSlots.init, fillAdvertSlots.customTiming],
-        ['cm-paidContainers', paidContainers.init]
+        ['cm-paidContainers', paidContainers.init],
+        ['cm-paidforBand', paidforBand.init]
     ];
 
     var customTimingModules = [];
 
-    if (config.page.isAdvertisementFeature) {
-        secondaryModules.push(['cm-paidforBand', paidforBand.init]);
```
Comment (katebee) [inline static/src/javascripts-legacy/bootstraps/commercial.js]: Out of interest, what is the reasoning behind remove the check and adding `cm-paidforBand` straight to secondary modules?

Comment (regiskuckaertz) [inline static/src/javascripts-legacy/bootstraps/commercial.js]: Cleaner code I guess. All commercial-related flags are in `commercial-features` and each module starts by checking if the corresponding flag is enabled.

Comment (katebee) [inline static/src/javascripts-legacy/bootstraps/commercial.js]: Cheers 😄

**Review by katebee — APPROVED** [review]
👍 LGTM

**Pull request merged**